client/package.json

```diff
@@ -57,7 +57,6 @@
     "daisyui": "^4.4.24",
     "eslint": "^8.56.0",
     "eslint-config-standard": "^17.1.0",
-    "eslint-plugin-import": "^2.29.1",
```

Comment:
why has this been removed?

Reply:
Hi @mendacium-a11y  ! That got removed with some other dependencies I removed. Thanks for pointing it out 😄. Added it now.